bundle/manifests/self-node-remediation.clusterserviceversion.yaml

```diff
@@ -36,7 +36,7 @@ metadata:
       ]
     capabilities: Basic Install
     categories: OpenShift Optional
-    containerImage: quay.io/medik8s/self-node-remediation-operator:0.5.0
+    containerImage: quay.io/medik8s/self-node-remediation-operator:latest
```

Comment:
If we set v0.0.1 in the images below then why we use `latest` in here?

Reply:
good catch, it had to do with order of declaration of the vars, fixed.

Reply:
Isn't it because of https://github.com/medik8s/self-node-remediation/pull/72/commits/aaa80da2b9472f4eb0b7ff3d4ce3e61f3cb0ac67#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L27?